### PR TITLE
Various grammer and typo nits

### PR DESCRIPTION
**Pull request by JAG-UK**
Fairly big conflict rebase of the reverted #80, but hopefully all good and simple.

### CODE REVIEW

File: draft-ietf-scitt-architecture.md
Comment (henkbirkholz) [inline draft-ietf-scitt-architecture.md:131]:
```suggestion
Trust in the Transparency Service itself is supported both by protecting their implementation (using, for instance, replication, trusted hardware, and remote attestation of a system's operational state) and by enabling independent audits of the correctness and consistency of its Registry, thereby holding the organization that operates it accountable.
```

Comment (henkbirkholz) [reply draft-ietf-scitt-architecture.md:131]: nitpicking

**Review by henkbirkholz — APPROVED** [review]
One non-blocking nit.

**Comment by henkbirkholz** [comment]
Thanks a ton, Jon, for the thorough review! :+1:

File: draft-ietf-scitt-architecture.md
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-architecture.md:134]:
```suggestion
Breadth of access is critical so the Transparency Service specified in this architecture cater to two types of audiences:
```

File: draft-ietf-scitt-architecture.md
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-architecture.md:195]: @JAG-UK I think software was deliberately added as the current scope was only `software`

Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md:195]: Sorry I just realized that software is already used in 194 so fine with it: now.

File: draft-ietf-scitt-architecture.md
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-architecture.md:517]:
```suggestion
Each Transparency Service also defines its own Registration Policies, which MUST apply to all entries in the Registry.
```

File: draft-ietf-scitt-architecture.md
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-architecture.md:1029]:
```suggestion
On one hand, this enables valid actors to detect and disambiguate malicious actors who issue contradictory Signed Statements to different entities (Verifiers, Auditors, Issuers), otherwise known as 'equivocation'.
```

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM! I have left few .nits, Please do consider before merging. 
You have my approval to merge

**Pull request merged**
